modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -81,9 +82,24 @@ def build_method_wrapper(name, method):
 
     @wraps(method)
     def method_wrapper(self, *args, **kwargs):
+        # If the method wasn't found in the 'BaseQueryCompiler' that means,
+        # that we're calling one that is OmniSci backend-specific, if we intend
+        # to fallback to pandas on 'NotImplementedError' then the call of this
+        # private method is the cause of some public QC method, that will catch
+        # arisen here exception and do fallback properly
+        default_method = getattr(BaseQueryCompiler, name, None)
```

Comment:
why do we assume that parent class is `BaseQueryCompiler`?.. this could be changed in the future, using `super()` feels better if that is possible (remember that `super()` can take up to two arguments to be independent of whatever method its call was made from).

Reply:
changed to retrieving the default method from the parent query compiler instead of the `BaseQueryCompiler`